### ISSUE TITLE
Neutral culture RESX File causes Build error

### ISSUE THREAD

**Issue by jobolo23** [dotnet/maui#5538]
### Description

When creating a Culture resource for a specific country e.g. 'Resources.de-DE.resx' and setting the culture of the Application to that everything works as expected but when creating a neutral Culture file e.g. 'Resource.de.resx'  the application doesn't build anymore. 

### Steps to Reproduce

1. Create new MAUI App
2. Add resource 'Resources.resx'
3. Add neutral Culture resource 'Resources.de.resx'

### Version with bug

Preview 14 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 14.2

### Did you find any workaround?

No

**Comment by jfversluis** [comment]
Could you please define "does not build anymore"? What are the build errors you see? What is any log output you get? Ideally, please add a little reproduction project. This information is too little to effectively diagnose any issue that might be there.

Thanks!

**Comment by jobolo23** [comment]
Hi I just found the issue and it was totally my bad. I copied the Resource File and renamed them to "Resource.de.resx" and "Resource.en.resx" but in the Resource.Designer.cs Files the Class waws automatically named Resource___Copy and when i then compiled it the output clearly showed that this was the issue. I now renamed the Classes in the Resource.Designer.cs and now everything is working as expected.

Sorry 😢
